joint_limits:
    right_front_wheel_joint:
      has_position_limits: false
      min_position: 0
      max_position: 0
      has_velocity_limits: true
      max_velocity: 1.80
      min_velocity: -1.80
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: false
      max_jerk: 0
      has_effort_limits: false
      max_effort: 0
      min_effort: 0

    right_rear_wheel_joint:
      has_position_limits: false
      min_position: 0
      max_position: 0
      has_velocity_limits: true
      max_velocity: 1.80
      min_velocity: -1.80
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: false
      max_jerk: 0
      has_effort_limits: false
      max_effort: 0
      min_effort: 0
    
    left_front_wheel_joint:
      has_position_limits: false
      min_position: 0
      max_position: 0
      has_velocity_limits: true
      max_velocity: 1.80
      min_velocity: -1.80
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: false
      max_jerk: 0
      has_effort_limits: false
      max_effort: 0
      min_effort: 0

    left_rear_wheel_joint:
      has_position_limits: false
      min_position: 0
      max_position: 0
      has_velocity_limits: true
      max_velocity: 1.80
      min_velocity: -1.80
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: false
      max_jerk: 0
      has_effort_limits: false
      max_effort: 0
      min_effort: 0
   
   
